src/sentry/utils/sdk.py

```diff
@@ -29,7 +29,7 @@
 
 # Can't import models in utils because utils should be the bottom of the food chain
 if TYPE_CHECKING:
-    from sentry_sdk._types import Event, Hint
+    from sentry_sdk.types import Event, Hint
```

Comment:
this doesn't need to live in the block any more

Reply:
fixed!